### PR TITLE
My books list

### PR DESCRIPTION
**Pull request by NatalieLuka**
- list of my books,
- link to bookdetails (isbn)
- bookdetailspage for chosen books

### CODE REVIEW

File: app/books/index.js [hunk 1]
```diff
@@ -1,27 +1,84 @@
...
       <Pressable
-        onPress={() => router.push("/books/scanner")}
-        style={({ pressed }) => [
-          globalStyles.button,
-          {
-            backgroundColor: pressed ? COLORS.primary : COLORS.secondary,
-          },
-        ]}
+        style={globalStyles.button}
+        onPress={() => router.push(`/books/${item.isbn}`)}
```
Comment (Staicy-S) [inline app/books/index.js:18]: Don't you need any special styling in the pressed state anymore? :)

File: app/index.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import { Redirect } from "expo-router";
+import { useUser } from "@clerk/clerk-expo";
 
 export default function IndexPage() {
-  return <Redirect href="(home)" />;
+  const { user } = useUser();
+
+  return <Redirect href={!user ? "(home)" : "/profile"} />;
```
Comment (Staicy-S) [inline app/index.js:7]: Did this solve your problem?

File: app/profile/index.js [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+
+  async function handleGetToken() {
+    const token = await getToken();
+    console.log(token);
+  }
+
   return (
     <>
+      <Text>Hello {user?.primaryEmailAddress?.emailAddress}</Text>
       <Text style={globalStyles.heading}>I am the Profilepage</Text>
```
Comment (Staicy-S) [inline app/profile/index.js:18]: Don't you need some content here 😝? Just joking

**Review by Staicy-S — APPROVED** [review]
Very nice!

File: app/index.js (hunk 2 above)
Comment (NatalieLuka) [reply app/index.js:7]: Yes it solved my problem.. Somehow.. 🤣

File: app/profile/index.js (hunk 3 above)
Comment (NatalieLuka) [reply app/profile/index.js:18]: ah.. who needs content ? unnecessary 🤌🏼

**Pull request merged**